coco/_version.py

```diff
@@ -56,7 +56,7 @@ class NotThisMethod(Exception):
 
 
 def register_vcs_handler(vcs, method):  # decorator
-    """Decorator to mark a method as the handler for a particular VCS."""
```

Comment:
I wouldn't change this file if possible as it's actually generated by `versioneer`. I imagine it will be easy for us to deal with `versioneer` upgrades etc. if we keep it as is.